src/resolve/resolved-varlink.c

```diff
@@ -516,6 +561,38 @@ static int vl_method_resolve_address(Varlink *link, JsonVariant *parameters, Var
         return 1;
 }
 
+#if ENABLE_VARLINK_NOTIFICATIONS
+static int vl_method_subscribe_dns_resolves(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata)
+{
+        Manager *m;
+
+        assert(link);
+
+        m = varlink_server_get_userdata(varlink_get_server(link));
+        assert(m);
+
+
+        if (json_variant_elements(parameters) > 0)
+                return varlink_error_invalid_parameter(link, parameters);
+
+        if (!FLAGS_SET(flags, VARLINK_METHOD_MORE))
+        {
+                // the client didn't set the more flag, so return an empty response and close the connection
+                return varlink_reply(link, NULL);
+        }
```

Comment:
please read up coding style...

https://systemd.io/CODING_STYLE/#formatting

i.e. no `//` comments please

`{` on same line as `if`

no `{}` around single-line if blocks.

Reply:
I believe the latest patch should adhere to the style guide now